### ISSUE TITLE
Fix to no warn `require-accessible-name` when the element has the parent possibly had a name

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#643]
- Markuplint version: `3.0.0`
- Parser lang: JSX
- Node.js version: `18`
- OS: macOS

## Describe the bug

Fix to no warn `require-accessible-name` when the element has the parent possibly had a name.

## Code Example

```html
<svg aria-label={label}>
  <rect />
</svg>
```

## Steps To Reproduce

1. Specify `@markuplint/jsx-parser`
2. Run

## The current behavior

```
Require accessible name (require-accessible-name)
```

to the `<rect />` element.

## The expected behavior

No warning.

**Comment by YusukeHirao** [comment]
It resolves that it **adds the `img` role** to the svg element.
